charts/jaeger/values.yaml

```diff
@@ -200,6 +200,9 @@ agent:
       # type: RollingUpdate
       # rollingUpdate:
       #   maxUnavailable: 1
+  # Set to false if your release is called `jaeger`.
+  # Read More: https://github.com/jaegertracing/jaeger/issues/1986
```

Comment:
https://github.com/jaegertracing/jaeger/issues/1986  concerns the query component was worked around via https://github.com/jaegertracing/helm-charts/pull/12, can you plz elaborate on what issue this solves or how to reproduce? Thanks!

Reply:
This is supposed to resolve a problem related to exposing jaeger service port via an env variable, causing major issues if the chart name is exactly "jaeger".